File: 01_ProgrammingBasics/_01_FirstStepsInProgramming/_06_ConcatenateData.java
package _01_ProgrammingBasics._01_FirstStepsInProgramming;

import java.util.Scanner;

public class _06_ConcatenateData {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input data
        String firstName = scan.nextLine();
        String lastName = scan.nextLine();
        int age = Integer.parseInt(scan.nextLine());
        String town = scan.nextLine();

        // Print formatted message
        System.out.printf("You are %s %s, a %d-years old person from %s.", firstName, lastName, age, town);
        
        scan.close();
    }
}

File: 23_C#Basics/_01_FirstStepsInProgramming/_06_ConcatenateData/_06_ConcatenateData.cs
using System;

public class _06_ConcatenateData
{
    public static void Main(string[] args)
    {
        // Read input data
        string firstName = Console.ReadLine();
        string lastName = Console.ReadLine();
        int age = int.Parse(Console.ReadLine());
        string town = Console.ReadLine();

        // Print formatted message
        Console.WriteLine($"You are {firstName} {lastName}, a {age}-years old person from {town}.");
    }
}
